competitions/cloud-cover/benchmark-tutorial.ipynb

```diff
@@ -0,0 +1,2586 @@
zDNFWRJSJHhPVdcyrypLPntWMk+Bf/Lz71iVBaVWjP3E/bHnxy9ucEbz4xfXnAbPf/rVG+YsORrruuLZ1Y6uG+mmme2q5dXdicPhRKk0P//2Lau6WF53aNqaJzdXgBFXiTXoaGiaGldYummmtAZdaw7JUOsM2uCnmXXlGGeJCZ36kZgTZSlzvWmasFbTDxMbV8vcPCcqp7HGkvxMjImUIwbJvTDO4X1knjwxClxBKRlHVa4ia8037/Z/Uwl7fPwLzeyUUl8Afx/4j4GnSyEk5/xKKfVk+bCXwH/00ad9s/zZb36tfwP4NwCePn1Cu71CpTVGKVZNKwPP1Ru+fXfmq/cjPSI9SQ8OC5C5ShZSA9qSjcItxAWjHKVx5LikuoelKC2lS6492TjIRSIJ8IqlhC0xgg8e14fgHfnhtajxl5mVJHTBw+HXoZgW0i4JjDPoZZa1axxdElS61mK0dkphs2LdFGgU49I5Xq9rmslx24/sqoJTPzOGSCDLpmuYmJLAMVUWeEJImdenDmfg092G3gee7VoKa1k1FRrNlwt55Z/87BWn0VNqTe8DyhhJerKWuixxVoFRPNk1hHnmu1e3EoY8BN4depYGTnR1Sbo7u4wKYkoCVlAy/yoLi1LgrF2eswzZ0+T35LvM5J9iiyuePK+oVi0XuzWr2kHwOBJTrsBZVNEsqvqMc/LaGAVaZZR+UHY/GBaWJYk2KJaUK7sGcybPEzl74nDEaNFIUoqw3SoDztAYzSkEUhb2nlWgicQQlt9H8X5/YlOXkkWSM3f3e3KUcYEdepSqMEoTvGDMV6uGw/GE0yJHmVKi72eKopIIQhMJc6Cqaqw1vH1zz+HUcbVp0dbiQ+SPfvWeT59csq5K/n9/+R23/cA/+r0vUEpz05Y8qUtev9vzi+07/t6Xn6KVoptn2qbCuoIvXz5lmgZMnChIDMPArqnYH05opfFFpChKnC0IIdD3nrqwzP2IUplpFvmQRm5266ahtop5mmicZeoHppipSsscAlVV4vtR5F5WTlbnSeyUm7og5yCQUat48exyoeZ4tC3ou5G7wwlrNZUraNoGhQjAqxhpqn8FUYpKqRXwvwf+Zznn48OZ/rd96G/5s78ic845/3vAvwfwk5/8JBfVmspK1+Xqlpc5E4s1yr7ifvyO6Sw+z4fNbF4kJpIVkfDjKMc+I4SUXelonBS20jnC6MlK5lpGfQjBfhi0qyzFVDqSB939gxhv+cW0HINTzniphRQLBl4rvejLFAVa0MhpAYsqqLVmDhFTFVxghZqSpMNZFU48tVZyLdqQ+fGTHcMcaJygtUtncUhaWciZcpmBNVY+t5s8KguOqIzyc35zPHN3mvivfPmcGCK/eHWLNYpA4O7gaRuxVoGhmQK3x56qKBY8EeicSD5xe3dEK8W+Dxy6DqVESgNyEzDox5ndwxP7cOQH9UiZ0UoWSWWxZHaEzOn9mW8pGV5csXtuaeJIHSpWTlGqSCSSqbC1ZlVm6tKhzSI3ygs1N+clhzYtC+CHxZVGZTnCkgOQqFZrxrFjCgkVTmjMEgKemMKMU0aG4kYAkzlJiLjSmXmK6ODxw8Bqs2aeA6uqoK5LOfoPI4djx7pydIOnbjyxKNBWjmNFVctz4++WRHjBlKW8oL9SlCjGnMgkxgdtY1MxzpknN5esNye+ubtj23W8vu+wRnP77sgf5V/yxbNr1m3NHzzf8Z/+4i1/9u17Ykhc71quLzZs1qKXqwqLMRtO+z1PdhKpeO4nrFKMXYeZLW5bkFXA5ITRcJ4m6uVGNseI8UEWGRlMVcq8LUykMBKWa6StHNqJ1bIoHNaADzKTPp3OXO1WGKvpjgNV5VBR4AQ5JgEPxEBhNau2oW1rckxYZ7m6aDHGcn/s0OZvXDs8Pr5XsVNKOaTQ/W9yzv+H5Y/fKKWeL13dc+Dt8uffAJ9+9OmfAN/9dV9fdFQFRVnJ99Ml5e4zLoZEH6F6fUD3RyAtA8ulO9OIsBRIKRDDQM6aTIRC2GnrumS3qjmd+sdjrHRh6lGD9ZA7q5ALNKuHY6l+/NjMh7mcT5mQZRv1IKt4FNZmiddzSpOtSGpEXxepjKXvZsrS8GLTEsi8PvSyw9WKGOD59Yo3+443+zNKKz673sIZvnp34Dh6+QkWZlmIkqlplGxrP92teLZtMQYmL8FBXz7dgUp8ezjzzf2Z59uWJ5uGyiq+eLKhsoavX++ZC83ZCscv+ZlVZUkpc+gmCYcG+lm4ZoU1CFA1LQufB9uYPAchSuBMUZXLQiLiYxABKTKKSCiwli5rjkmz1gaTM0FvmKNG55HkLTFJTqzVBpszTkfp8JeZGiotub+yxf4okWQRcOblmJgkt8AqqtoRJvBR45Pl1I1UZiJ4jyfTnTuy0jRVIeOTLPPhBwabS4kwjcR5FhgmooE83p+Y54guHYVR5BBkjlhYVFbM40DX9fTdSOG0dIxZk1QUwe04CZZMKYauJ+XMatVSNw3HzhNoePqjf43efUea3pJyTVVCreGX394xjImnu5qcFDerhvvJcxoH1pNjngPOGKLSRBS2MGAdo+84TYEQEoXSKGdwRqHijI9axkPWorX8npP3FCaRfSIGUSyMKRC1om1bCDP1EoKeRClD4bSQgBScu47SR1ZtRc4wTYGYEnVTUi6BSroqiIvTKclGjf3dmayhdJph9Kzbhk37AFz4fo+/sdgpaeH+V8Cf5pz/3Y/+6v8M/A+Bf2f55//poz//3yql/l1kQfG7wH/y130PbTSubDCugCywPmtqVusrysPItim53Ws8PIqOdYYY5W2ttEJUXhkfA9YohuC57QbWTcmzqzXH88Dp2JGyMOBURjJDl4tCLg8JblRKP15QKmeSyo8AgOAzIS3aLhb5yuL1REmxjCmhFj2eNYY+BNCaalFkmKS4P0/MC704hsToA7W1zHNgjoJ62jQlb889392fqZxj9Iln65r3/biE0SxFH1hXjqtVjbGKdVkyho6LumLX1vgo
```

Comment:
I'm seeing an `AttributeError: 'functools.partial' object has no attribute '__name__'` on this line when running this on the (staging) hub. Do you see it in prod?

IIRC this was from some incompatibility between some pytorch library and pandas. Do you recall how we solved it in the last notebook?

Reply:
I don't see this in prod (mine runs fine), but I have run into that error before and can't remember exactly how I fixed it. What are you versions of pytorch and pandas?